src/test/soundproxy_test.cpp

```diff
@@ -21,10 +21,10 @@ class SoundSourceProxyTest : public MixxxTest {
 TEST_F(SoundSourceProxyTest, ProxyCanOpen) {
     // This test piggy-backs off of the cover-test files.
     const QString kCoverFilePath(
-            QDir::currentPath() + "/src/test/id3-test-data/cover-test.");
+            QDir::currentPath() + "/src/test/id3-test-data/cover-test");
 
     QStringList extensions;
-    extensions << "aiff" << "flac" << "mp3" << "ogg" << "wav";
+    extensions << ".aiff" << ".flac" << "-png.mp3" << ".ogg" << ".wav";
```

Comment:
for 2 reasons:
1. I didn't want to modify soundproxy test, cause I'm not sure to understand what (and how) it currently does
2. from a "sound" point of view, they are the exact same files. Only the embedded cover differ, so this is probably not a significant test to add for soundproxy.